src/Protocol/Diaspora.php

```diff
@@ -1117,6 +1122,20 @@ private static function contactByHandle($uid, $handle)
 		return $contact;
 	}
 
+	/**
+	 * Checks if the given contact url does support ActivityPub
+	 *
+	 * @param string  $url    profile url
+	 * @param boolean $update Update the profile
+	 * @return boolean
+	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
+	 * @throws \ImagickException
+	 */
+	public static function isSupportedByContactUrl($url, $update = null)
```

Comment:
Please change the default value for `$update` to `false` since it's a boolean and there's no check for `null` in the function body.

Reply:
Same as above.